signupValidation.php

```diff
@@ -0,0 +1,48 @@
+<?php 
+
+const DS = DIRECTORY_SEPARATOR;
+require_once ".". DS . 'config' . DS . 'dbconfig.php';
+
+if(isset($_GET['login'])&& $_GET['login']!=''){
+    // Getting the login that is in the input
+    $login = $_GET['login'];
+    // Connecting to DB
+    $conn = new mysqli(DB_SERVER_NAME, DB_USER, DB_PASS, DB_NAME);
+    // Error msg
+    if ($conn->connect_error) {
+        die("Connection failed: " . $conn->connect_error);
+    }
+    // Query 
+    $sql = "SELECT login
+                from users
+                where login = " . $login ;
+    
+    $result = $conn->query($sql);
+    // Result in case to find something
+    if ($result->num_rows > 0) {
+        echo "This user name is being used";
+    } 
```

Comment:
Rewrite this code in this way:
https://www.w3schools.com/php/php_mysql_prepared_statements.asp